lib/browser/chrome-extension.js

```diff
@@ -215,6 +226,23 @@ ipcMainInternal.handle('CHROME_TABS_SEND_MESSAGE', async function (event, tabId,
   return ipcMainUtils.invokeInWebContents(contents, true, `CHROME_RUNTIME_ONMESSAGE_${extensionId}`, senderTabId, message)
 })
 
+ipcMainInternal.handle('CHROME_TABS_RELOAD', async function (event, tabId, extensionId, reloadProperties) {
+  const contents = webContents.fromId(tabId)
+
+  if (!contents) {
+    throw new Error(`Sending message to unknown tab ${tabId}`)
+  }
+  const shouldByPassCache = !!reloadProperties.bypassCache
+
+  if (shouldByPassCache) {
+    contents.reloadIgnoringCache()
+  } else {
+    contents.reload()
+  }
+
+  return true
```

Comment:
the `responseCallback` does not seem to have any arguments, no need to return anything here

Reply:
removed it